rclcpp/include/rclcpp/timer.hpp

```diff
@@ -71,10 +82,15 @@ class TimerBase
   bool
   is_canceled();
 
+  /// Reset the timer.
+  /**
+   * \throws std::runtime_error if the rcl_timer_reset returns a failure
+   */
   RCLCPP_PUBLIC
   void
   reset();
 
+  /// Handles the function to execute when the timer is emit a trigger.
```

Comment:
This calls the callback function when the timer signal is emitted right? It doesn't handle it